### ISSUE TITLE
Error while doing a plain build (Neo4j - Browser ... FAILURE)

### ISSUE THREAD

**Issue by telmotrooper** [neo4j/neo4j#6261]
I've been having issues while trying to build Neo4j with "mvn clean install", if I skip the tests with "mvn clean install -DskipTests" I don't get any errors at all. I'm using Antergos (an Arch based distro) and I downloaded all the dependencies I could find, even the ones that were only available through AUR. I think the only two I couldn't find were "rpm" and "nodejs-legacy", could that be causing the issue? I tried googling, but I didn't see anyone having the exact same problem I'm having. Someone suggest updating to the latest version of surefire, but I just downloaded Neo4j and Maven, I'm pretty sure everything is updated.

[INFO] Neo4j - Browser .................................... FAILURE [03:24 min]
[INFO] Neo4j - Server ..................................... SKIPPED
[INFO] Neo4j - Test Harness ............................... SKIPPED
[INFO] Neo4j - Server Plugin Tests ........................ SKIPPED
[INFO] Neo4j - Server Examples ............................ SKIPPED
[INFO] Neo4j - Cypher Acceptance .......................... SKIPPED
[INFO] Neo4j - Cypher Compatibility Suite ................. SKIPPED
[INFO] Neo4j - Community Build ............................ SKIPPED
[INFO] Neo4j - Graph DB Monitoring and Management tools ... SKIPPED
[INFO] Neo4j - Advanced ................................... SKIPPED
[INFO] Neo4j - Advanced Server ............................ SKIPPED
[INFO] Neo4j - Advanced Build ............................. SKIPPED
[INFO] Neo4j - Query Logging .............................. SKIPPED
[INFO] Neo4j - Enterprise Graph Database Kernel Features .. SKIPPED
[INFO] Neo4j - Communication Package ...................... SKIPPED
[INFO] Neo4j - Clustering Infrastructure .................. SKIPPED
[INFO] Neo4j - Backup Tool ................................ SKIPPED
[INFO] Neo4j - High Availability .......................... SKIPPED
[INFO] Neo4j - Core Edge .................................. SKIPPED
[INFO] Neo4j - Metrics Kernel Extension ................... SKIPPED
[INFO] Neo4j - Enterprise ................................. SKIPPED
[INFO] Neo4j - Enterprise Server .......................... SKIPPED
[INFO] Neo4j - Enterprise Performance Tests ............... SKIPPED
[INFO] Neo4j - Enterprise Test Harness .................... SKIPPED
[INFO] Neo4j - Enterprise Build ........................... SKIPPED
[INFO] Neo4j - Integration Tests .......................... SKIPPED
[INFO] Neo4j - Stress Tests ............................... SKIPPED
[INFO] Neo4j - Low Level Tools ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:46 h
[INFO] Finished at: 2016-01-19T19:50:05-02:00
[INFO] Final Memory: 73M/367M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.15:test (default-test) on project neo4j-browser: Error occurred in starting fork, check output in log -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :neo4j-browser

**Comment by oskarhane** [comment]
Hi @telmotrooper, thanks for the report.  
Yes, it's definitely the missing `nodejs-legacy` part that breaks the build.  

Would you mind pulling the latest code and trying again?  
We just (13 hours ago) removed the browser build process and pull it as a dependency instead so `nodejs-legacy` isn't required anymore.

**Comment by telmotrooper** [comment]
@oskarhane Thank you for your answer. I was able to build a plain build (mvn clean install) with the latest code, but it also introduced a new, worse, issue. Now I can't access the browser when I open the server, the page looks all messed up. It happens whenever I make a build, I tried recloning the repository quite a few times.

Screenshot: http://i.imgur.com/RIOW1Ry.png

Should I open a new issue? For now I'll close this one since I'm able to do a plain build.
